socorro/cron/jobs/bugzilla.py

```diff
@@ -80,12 +80,6 @@ def find_signatures(content):
     return signatures
 
 
-class NothingUsefulHappened(Exception):
-    """an exception to be raised when a pass through the inner loop has
-    done nothing useful and we wish to induce a transaction rollback"""
-    abandon_transaction = True
```

Comment:
This wasn't used anywhere, so I nixed it. It's the only thing that had a `abandon_transaction` property.